cookbooks/bootstrap/.rubocop.yml

```diff
@@ -0,0 +1,31 @@
+AllCops:
```

Comment:
Was this cargo-culted from somewhere? Seems kinda like overkill. Perhaps the ChefStyle one?